spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -624,22 +661,31 @@ function recvPacket(
 
     // all assertions passed (except sequence check), we can alter state
 
```

Comment:
```suggestion
    // assume the packet is successfully received 
    packetReceipt = SUCCESSFUL_RECEIPT
```

Reply:
The packet is successfully received at this point. There's no assumption being made

Reply:
What I meant by the comment is that we set a variable to `SUCCESSFUL_RECEIPT` to avoid multiple if conditions below, i.e.,`packetReceipt =  SUCCESSFUL_RECEIPT` unless stated otherwise.